modules/UI/recording/Recording.js

```diff
@@ -452,12 +450,13 @@ const Recording = {
     },
 
     // checks whether recording is enabled and whether we have params
-    // to start automatically recording
+    // to start automatically recording (XXX: No, it doesn't do that).
```

Comment:
I agree that this would be better (and even better would be to refactor the code), but I don't understand the code and it is outside the scope of my current work to understand it. I added the note in the hope that it will remind/nudge the next person who actually works with this code to fix it.